WordPressKit/WordPressKit/PluginDirectoryEntry.swift

```diff
@@ -151,27 +151,23 @@ extension PluginDirectoryEntry: Decodable {
 typealias Author = (name: String, link: URL?)
 
 private func extractAuthor(_ author: String) -> Author {
+    let name = author.replacingOccurrences(of: "<[^>]+>", with: "", options: String.CompareOptions.regularExpression, range: nil)
```

Comment:
Search helped me find an edge-case where the previous approach failed miserably. It's much simpler now and!